common/djangoapps/course_modes/tests/test_models.py

```diff
@@ -150,11 +150,18 @@ def test_course_has_payment_options(self):
         honor.save()
         self.assertTrue(CourseMode.has_payment_options(self.course_key))
 
+    def test_course_has_payment_options_with_no_id_professional(self):
+        # Has payment options.
+        _, _ = self.create_mode('no-id-professional', 'no-id-professional', min_price=5)
```

Comment:
You don't need the `_, _` part, right? You can just use `self.create_mode(...)` by itself without using the return values.